README.md

```diff
@@ -3,88 +3,108 @@
 
 These instructions have been used on the following operating systems.
 
-* Linux
-* Mac OS
-* Windows 10 Pro
-
-Please note that Windows 10 Home is not supported by Docker Desktop at this time.
+- Linux
+- Mac OS
+- Windows 10 Pro - Please note that Windows 10 Home is not supported by Docker Desktop at this time.
 
-1. Fork this repository. This creates a copy of the repository for you to work on. For more help see this GitHub guide: [Fork a repo](https://help.github.com/en/github/getting-started-with-github/fork-a-repo).
-2. Clone your fork. This creates a copy on your local computer. For more help see this GitHub guide: [Cloning a repository](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository).
+1. Fork this repository. This creates a copy of the repository for you to work on. For more help see this GitHub guide: [Fork a repo].
+2. Clone your fork. This creates a copy on your local computer. For more help see this GitHub guide: [Cloning a repository].
 
-```plain
-git clone git@github.com:codebuddies/django-concept.git cb
+```bash
+$ git clone https://github.com/codebuddies/backend codebuddies-backend
 ```
 
 3. Navigate into the project directory.
 
-```plain
-cd cb
+```bash
+$ cd codebuddies-backend
 ```
 
 4. Start the local development environment.
 
-```plain
-docker-compose up -d
+```bash
+$ docker-compose up -d
 ```
 
+**Note:** `-d` starts Docker in detatched mode. See [logs](#debugging-with-docker-logs)
+
+### Local development environment details
+
 This will run the following components:
 
-- Nginx, a web server: http://localhost:8000
-- Adminer, a DB front-end: http://localhost:8001
+- Nginx, a web server: http://localhost:8000 - view the application or make API calls
 - Mailhog, a dummy mailbox: http://localhost:8025
-- A PostgreSQL database
 - The Django web application
+- Adminer, a DB front-end: http://localhost:8001
+- A PostgreSQL database `postgres://babyyoda:mysecretpassword@localhost:5432/codebuddies`
+
+You can access the database through the Adminer front-end or using a local PostgreSQL client
 
-You can view the application or make API calls by using the Nginx URL.
+![screenshot of Adminer](https://i.imgur.com/Dtg5Yel.png =250x)
 
-You can access the database through the Adminer front-end or using a local PostgreSQL client and the following URL: `postgres://babyyoda:mysecretpassword@localhost:5432/codebuddies`.
+5. Create a superuser so that you can log into `http://localhost:8000/admin` by running the following in your terminal: 
 
-![screenshot of Adminer](https://i.imgur.com/Dtg5Yel.png)
+```bash
+$ docker-compose run --rm app ./manage.py createsuperuser
+```
 
-To stop the application and remove all containers, run the following.
+6. You can populate the database with some random test data for development purposes by running 
```

Comment:
Ohh, one idea here: maybe it would be helpful to add @chris48s' documentation about the `init_data --help` command: 

```
$ docker-compose run --rm app ./manage.py init_data --help

usage: manage.py init_data [-h] [--clear-db] [--num-users NUM-USERS]
                           [--num-tags NUM-TAGS]
                           [--num-resources NUM-RESOURCES] [--version]
                           [-v {0,1,2,3}] [--settings SETTINGS]
                           [--pythonpath PYTHONPATH] [--traceback]
                           [--no-color] [--force-color]

Initialize the DB with some random fake data for testing and development

optional arguments:
  --clear-db            Clear existing data from the DB before creating test
                        data
  --num-users NUM-USERS
                        Number of `User` objects to create (default 10)
  --num-tags NUM-TAGS   Number of `Tag` objects to create (default 10)
  --num-resources NUM-RESOURCES
                        Number of `Resource` objects to create (default 10)
  -v {0,1,2,3}, --verbosity {0,1,2,3}
                        Verbosity level; 0=minimal output, 1=normal output,
                        2=verbose output, 3=very verbose output
```

This is from his PR at https://github.com/codebuddies/backend/pull/129

Reply:
Done